deploy/kubernetes/weavescope.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
```

Comment:
Is this supposed to be here?

I don't think it'll work with fluxy, which expects either Deployment or ReplicationController resources. But it might not break it.